src/cascadia/TerminalSettingsEditor/Appearances.cpp

```diff
@@ -915,6 +935,41 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         LightColorSchemeName(val.Name());
     }
 
+#define GET_COLOR_PREVIEW(appearanceVal, deducedVal)     \
```

Comment:
FWIW, this macro could just be a function! you can take all the arguments by `&` of course to ensure they don't get copied. It probably will get inlined into each function too. Benefits include proper type checking and syntax highlighting